src/download-helpers.js

```diff
@@ -209,8 +209,9 @@ const basicAuthentication = async (page, overridePage, url, username, password,
   await page.type('[data-test-subj="password"]', password);
   await page.click('button[type=submit]');
   await page.waitForTimeout(10000);
+  const tenantSelection = await page.$("h4::-p-text(Select your tenant)");
   try {
-    if (multitenancy === true) {
+    if (multitenancy === true && tenantSelection !== null) {
```

Comment:
if tenant selection is null, is it possible that the `Select your tenant` popup didn't show up but user is still in a wrong tenant?

for example the dashboard is in global tenant but after login, user is in private tenant and the `Select your tenant` didn't show up, would that happen?

Reply:
That shouldn't happen when report is generated with short URL with tenant info. e.g. `https://<domain>.us-west-2.on.aws/_dashboards/goto/8fa30d307ea81f81a5edc692e189c297?security_tenant=global`